app/code/core/Mage/Page/etc/system.xml

```diff
@@ -82,22 +82,22 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </default_robots>
-<!--
-                        <default_media_type translate="label">
-                            <label>Default Media Type</label>
-                            <sort_order>50</sort_order>
-                            <show_in_default>1</show_in_default>
-                            <show_in_website>1</show_in_website>
-                            <show_in_store>1</show_in_store>
-                        </default_media_type>
-                        <default_charset translate="label">
-                            <label>Default Charset</label>
-                            <sort_order>60</sort_order>
-                            <show_in_default>1</show_in_default>
-                            <show_in_website>1</show_in_website>
-                            <show_in_store>1</show_in_store>
-                        </default_charset>
--->
+                        <!--
+                                                <default_media_type translate="label">
+                                                    <label>Default Media Type</label>
+                                                    <sort_order>50</sort_order>
+                                                    <show_in_default>1</show_in_default>
+                                                    <show_in_website>1</show_in_website>
+                                                    <show_in_store>1</show_in_store>
+                                                </default_media_type>
+                                                <default_charset translate="label">
+                                                    <label>Default Charset</label>
+                                                    <sort_order>60</sort_order>
+                                                    <show_in_default>1</show_in_default>
+                                                    <show_in_website>1</show_in_website>
+                                                    <show_in_store>1</show_in_store>
+                                                </default_charset>
+                        -->
```

Comment:
This commented block has an issues. Every line has more space at the left side. This need to be fixed.

Reply:
Guess it was automatically changed with phpcs/phpcbf. Maybe remove it at all?

Reply:
I don't know what happened there. I have deleted the comment.